### ISSUE TITLE
Don't reopen the store

### ISSUE THREAD

**Issue by runspired** [funkensturm/ember-local-storage#367]
this reopen is largely unnecesssary as it is, just adding to the class directly would work though even better would be to keep these as utils only.

**Comment by fsmanuel** [comment]
@runspired thanks for opening the issue. I'll look into it.

**Comment by fsmanuel** [comment]
@runspired you are talking about this, right?

https://github.com/funkensturm/ember-local-storage/blob/aa1f8377b028af7cc40a7fba5d99f44a22834971/addon/initializers/local-storage-adapter.js#L7-L19

I was already planning to provide it as utility functions. So I'll deprecate it and remove it in a furtue version.

**Comment by mwpastore** [comment]
I'm getting the following error after doing an ember-cli-update to 4.12:

```
local-storage-adapter.js:11
Uncaught TypeError: _emberData.default.Store.reopen is not a function
    at Object.initialize (local-storage-adapter.js:11:1)
    at index.js:130:1
    at Vertices.each (dag-map.js:192:1)
    at Vertices.walk (dag-map.js:121:1)
    at DAG.each (dag-map.js:66:1)
    at DAG.topsort (dag-map.js:72:1)
    at App._runInitializer (index.js:154:1)
    at App.runInitializers (index.js:128:1)
    at App._bootSync (index.js:489:1)
    at App.domReady (index.js:390:1)
```

**Comment by fsmanuel** [comment]
@runspired can you tell me which version of ember data removed the `Store.reopen`? From the failing tests I would say it's `4.12` is this correct?

**Comment by fsmanuel** [comment]
Released in [v2.0.5](https://github.com/funkensturm/ember-local-storage/releases/tag/v2.0.5)

**Comment by fsmanuel** [comment]
@runspired Thanks for the confirmation! I'll have to get more familiar with the changes in ember data and how to utilize them here. From what I saw it looks like a lot of work with great results. Thanks for working so hard on ember data!
